ovn-tester/ovn_tester.py

```diff
@@ -118,7 +127,21 @@ def read_config(configuration):
         bringup_cfg = ClusterBringupCfg(
             n_pods_per_node=bringup_args.get('n_pods_per_node', 10)
         )
-        return log_cmds, cluster_cfg, brex_cfg, bringup_cfg
+
+        netpol_multitenant_args = config.get('netpol_multitenant', dict())
+        netpol_multitenant_cfg = NsMultitenantCfg(
+            n_namespaces=netpol_multitenant_args.get('n_namespaces', 0),
+            n_external_ips1=netpol_multitenant_args.get('n_external_ips1', 3),
+            n_external_ips2=netpol_multitenant_args.get('n_external_ips2', 20),
+            ranges=[
+                NsRange(
+                    start=range_args.get('start', 0),
+                    n_pods=range_args.get('n_pods', 5),
+                ) for range_args in netpol_multitenant_args.get('ranges')
```

Comment:
What happens here if `netpol_multenant_args` has no `ranges` configured? I expect that `netpol_multitenant_args.get()` will return `None`, resulting in a `TypeError`.

Reply:
Fixed, thanks for spotting this!